src/components/AboutUsIntroduction/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
```

Comment:
Please use `.tsx` for file type.

Reply:
@Kittinske15 This PR is successfully built ka. That's issue occurs in `feature/home-search-branch`

<img width="938" alt="Screenshot 2564-03-01 at 00 51 25" src="https://user-images.githubusercontent.com/32285705/109428143-42077e80-7a28-11eb-97e2-892adb3140f8.png">